packages/ui/src/primitives/tailwind/Checkbox/index.tsx

```diff
@@ -38,35 +40,23 @@ export interface CheckboxProps extends Omit<React.InputHTMLAttributes<HTMLInputE
 }
 
 const Checkbox = ({ className, containerClassName, label, value, onChange, disabled, ...rest }: CheckboxProps) => {
-  const twClassName = twMerge(
-    'disabled:border-steel-400 disabled:bg-steel-400 peer',
-    'relative h-4 w-4 shrink-0 appearance-none rounded-sm',
-    'border-2 border-blue-500 bg-white',
-    'checked:border-0 checked:bg-blue-800 focus:outline-none focus:ring-2',
-    'focus:ring-blue-500 focus:ring-offset-0 dark:focus:ring-blue-600',
-    className
-  )
-
   return (
-    <div className={twMerge('flex w-full items-center gap-4', containerClassName)}>
-      <input type="checkbox" onChange={(e) => onChange(e.target.value as any)} className={twClassName} {...rest} />
-      {label && (
-        <Label onClick={() => onChange(!value)} className="cursor-pointer self-stretch">
-          {label}
-        </Label>
+    <div
+      className={twMerge(
+        'grid h-[16px] w-[16px] cursor-pointer place-items-center rounded border border-theme-primary',
```

Comment:
we should use tailwind relative sizes. in this case it should be `h-4 w-4`

Reply:
Done!